matchers.go

```diff
@@ -37,6 +38,13 @@ func PrefixMatcher(strs ...string) Matcher {
 	return pt.matchPrefix
 }
 
+// PrefixByteMatcher returns a matcher that matches a connection if it
+// starts with any of the []byte in list.
+func PrefixByteMatcher(list ...[]byte) Matcher {
```

Comment:
no, I just did it to match the other PrefixMatcher. I can unexport if you prefer